<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CWizFrameSet</class>
 <widget class="QWizard" name="CWizFrameSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>410</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Image Set Wizard</string>
  </property>
  <property name="windowIcon">
   <iconset resource="lgck-builder.qrc">
    <normaloff>:/images/lgckbuilder64ronnie2.png</normaloff>:/images/lgckbuilder64ronnie2.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ClassicStyle</enum>
  </property>
  <property name="options">
   <set>QWizard::IndependentPages</set>
  </property>
  <property name="titleFormat">
   <enum>Qt::AutoText</enum>
  </property>
  <widget class="QWizardPage" name="pageIntro">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="lgck-builder.qrc">:/images/bitmap1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="lgck-builder.qrc">:/images/bitmap1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>This wizard will guide you the process of importing images.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="sWelcome">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Welcome to the Image Set Wizard</string>
    </property>
   </widget>
  </widget>
  <widget class="CWizPageImport" name="pageGeneral">
   <property name="title">
    <string/>
   </property>
   <property name="subTitle">
    <string>Images</string>
   </property>
   <widget class="QToolButton" name="btnDeleteImage">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>223</y>
      <width>20</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Delete Image</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="lgck-builder.qrc">
      <normaloff>:/images/minus.png</normaloff>:/images/minus.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="btnAddImages">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>223</y>
      <width>25</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Add New Images</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="lgck-builder.qrc">
      <normaloff>:/images/plus.png</normaloff>:/images/plus.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>73</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Use the + and - button to add and removed images. You can also drag &amp; drop image in the box to add them. Right-click individual images for more options.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="CWFrameSet" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>63</y>
      <width>161</width>
      <height>161</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="indentation">
     <number>21</number>
    </property>
    <property name="rootIsDecorated">
     <bool>false</bool>
    </property>
    <attribute name="headerVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="eName">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>13</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>This is the human readable name for the object</string>
    </property>
    <property name="maxLength">
     <number>31</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>52</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>This image set can be edited later.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>btnDeleteImage</zorder>
   <zorder>btnAddImages</zorder>
   <zorder>label_8</zorder>
   <zorder>eName</zorder>
   <zorder>label_4</zorder>
   <zorder>label_29</zorder>
   <zorder>treeWidget</zorder>
  </widget>
  <widget class="QWizardPage" name="pageFinal">
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="lgck-builder.qrc">:/images/bitmap1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Congratulation, you have just created an image set. Click to the finish button to complete the process.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CWFrameSet</class>
   <extends>QTreeWidget</extends>
   <header>WFrameSet.h</header>
  </customwidget>
  <customwidget>
   <class>CWizPageImport</class>
   <extends>QWizardPage</extends>
   <header>WizPageImport.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="lgck-builder.qrc"/>
 </resources>
 <connections/>
</ui>
